### ISSUE TITLE
[FEATURE MODIFICATION] Rendy/gfx backend log verbosity

### ISSUE THREAD

**Issue by bamling** [amethyst/amethyst#1652]
## Feature

Logging for Rendy and the chosen gfx backend.

## Change

The log level for Rendy and the chose gfx backend (e.g. Vulkan) should by default be set to a less verbose level (`amethyst::LogLevelFilter::Warn` or `amethyst::LogLevelFilter::Error`).

## Reason

The current logs are a bit overly verbose and make it harder to find actual errors in your application. It's possible to configure a less verbose level yourself but that requires the following block of code:

```rust
amethyst::Logger::from_config(Default::default())
    .level_for("gfx_backend_vulkan", amethyst::LogLevelFilter::Warn)
    .level_for("rendy_factory::factory", amethyst::LogLevelFilter::Warn)
    .level_for(
        "rendy_memory::allocator::dynamic",
        amethyst::LogLevelFilter::Warn,
    )
    .level_for(
        "rendy_graph::node::render::pass",
        amethyst::LogLevelFilter::Warn,
    )
    .level_for("rendy_graph::node::present", amethyst::LogLevelFilter::Warn)
    .level_for("rendy_graph::graph", amethyst::LogLevelFilter::Warn)
    .level_for(
        "rendy_memory::allocator::linear",
        amethyst::LogLevelFilter::Warn,
    )
    .level_for("rendy_wsi", amethyst::LogLevelFilter::Warn)
    .start();
```

That's a lot of boiler plate just to achieve more readable logs.

**Comment by silverweed** [comment]
I would like to contribute to this issue.

**Comment by distransient** [comment]
Fixed by #2048
